docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,708 @@
+    "\n",
+    "$$H(\\mathbf{q}) = H_{gc}(\\mathbf{q}_{cf}) + H_{ch}(\\mathbf{q}_{cf}) + H_{in}(\\mathbf{q}) $$\n",
+    "\n",
+    "where \n",
+    "- $H_{gc}$ is the geometrical constrain term (governing the growth of the primary sequence of aminoacids without bifurcations\n",
+    "- $H_{ch}$ is the chirality constrain (enforcing the righ stereochemistry for the system)\n",
+    "- $H_{in}$ is the interaction energy terms of the system. In our case we consider only nearest neighbor interactions. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62d99a5f",
+   "metadata": {},
+   "source": [
+    "Further details about the used model and the encoding of the problem can be found in [2].\n",
+    "\n",
+    "<font size='2'>[2] A.Robert, P.Barkoutsos, S.Woerner and I.Tavernelli, Resource-efficient quantum algorithm for protein folding, NPJ Quantum Information, 2021 </font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "8f541d04",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_nature.problems.sampling.protein_folding.interactions.random_interaction import RandomInteraction \n",
+    "from qiskit_nature.problems.sampling.protein_folding.interactions.miyazawa_jernigan_interaction import MiyazawaJerniganInteraction \n",
+    "from qiskit_nature.problems.sampling.protein_folding.peptide.peptide import Peptide\n",
+    "from qiskit_nature.problems.sampling.protein_folding.protein_folding_problem import ProteinFoldingProblem\n",
+    "\n",
+    "from qiskit_nature.problems.sampling.protein_folding.penalty_parameters import PenaltyParameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fced9189",
+   "metadata": {},
+   "source": [
+    "### Protein Main Chain"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "350c031c",
+   "metadata": {},
+   "source": [
+    "The Protein consists of a main chain that is a linear chain of aminoacids. For the naming of different residues we use the one-letter code as defined in Reference [3]. Further details about the naming and the type of aminoacids can also be found in [4].\n",
+    "\n",
+    "For this particular case we demonstrate the generation of the qubit operator in a neuropeptide with the main chain consisting of 7 aminoacids with letter codes APRLFY (see also [2]).\n",
+    "\n",
+    "<font size=\"2\">[3] IUPAC–IUB Commission on Biochemical Nomenclature (1972). \"A one-letter notation for amino acid sequences\". Pure and Applied Chemistry. 31 (4): 641–645. doi:10.1351/pac197231040639. PMID 5080161.</font> <br>\n",
+    "<font size=\"2\">[4] https://en.wikipedia.org/wiki/Amino_acid</font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bf140d37",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "main_chain_len = 7\n",
+    "main_chain_residue_seq = [\"A\", \"P\", \"R\", \"L\", \"R\", \"F\", \"Y\"]"
```

Comment:
You no longer need to specify these separately (#309). I suggest to simply write:
```suggestion
    "main_chain = \"APRLRFY\"",
```